Sources/WeatherSDK/Endpoints/EndpointParams.swift

```diff
@@ -0,0 +1,27 @@
+// Code generated by smithy-swift-codegen. DO NOT EDIT!
+
+import class ClientRuntime.EndpointsRequestContext
```

Comment:
The WeatherSDK code generation changes give an idea of what is different after this change.  No further comment on WeatherSDK files.